src/enum.ts

```diff
@@ -7,6 +7,22 @@ export enum Pages {
   home = 'home',
   favorite = 'favorite',
   user = 'user',
+  history = 'history',
+  profile = 'profile',
+  notifications = 'notifications',
   pageNotFound = 'page-not-found',
   login = 'login',
+  friends = 'friends',
+  message = 'message',
+  groups = 'groups',
+  video = 'video',
+}
+
+export enum PERMISSIONS {
+  DELETE = 'delete:users',
+  READ = 'read:users',
+  WRITE = 'write:users',
+  CREATE = 'create:users',
+  EDIT = 'edit:users',
+  UPDATE = 'update:user',
```

Comment:
The scope of the permission differs from the the scope of the other permissions. `user` in this case.

Reply:
fixed